### ISSUE TITLE
To do NLP 7/4

### ISSUE THREAD

**Issue by xuanyuzhou98** [xuanyuzhou98/OnDeviceTrain#13]
ODT Todo:
1. Train in GPU with batch norms and then delete batch norm to finetune, check accuracy. (@tianrengao  )
2. Implement model weights loading from PyTorch (@xuanyuzhou98 )
3. 2, 2, 2. Train on android phone, reduce 200 to 20. Find a quick training recipe, better optimizer? (@floraxue )
4. Check 18 18 18, default gc frequency vs "decreased "gc frequency (@floraxue @tianrengao )

NLP Todo:
1. Engineering Todo: 
- 把waveglow的layers一层层转写到deeplearning4j
- 在gpu上复现waveglow (@BohanZhai )
2. Research Todo: 
- 把sample efficient 和 waveglow merge起来 
- Design sample efficient training pipeline
- Hopefully, we can directly delete text
